src/plugins/intel_cpu/src/config.h

```diff
@@ -63,8 +64,18 @@ struct Config {
     std::string verbose;
     std::string blobDumpDir = "mkldnn_dump";
     FORMAT blobDumpFormat = FORMAT::TEXT;
+    // std::hash<std::underlying_type<FILTER>::type> is necessary for Ubuntu-16.04 (gcc-5.4 and defect in C++11 standart)
+    std::unordered_map<FILTER, std::string, std::hash<std::underlying_type<FILTER>::type>> blobDumpFilters;
+
+    enum FUSING_FILTER {
+        BY_TARGET_TYPE,
+        BY_FUSED_TYPE,
+        BY_TARGET_NAME,
+        BY_FUSED_NAME,
+    };
+
     // std::hash<int> is necessary for Ubuntu-16.04 (gcc-5.4 and defect in C++11 standart)
-    std::unordered_map<FILTER, std::string, std::hash<int>> blobDumpFilters;
+    std::unordered_map<FUSING_FILTER, std::string, std::hash<int>> fusingFilters;
```

Comment:
Could you please elaborate on why different hash functions are used for blobDumpFilters and fusingFilters maps? What is the difference? In the first case `std::underlying_type` is used, but for the second `std::hash<int>` is used.

Reply:
No real reason.
Agree, that should be aligned.
Updated.